src/main/com/github/perlundq/yajsync/session/Sender.java

```diff
@@ -276,7 +285,8 @@ public Boolean call() throws ChannelException, InterruptedException
             }
             long t3 = System.currentTimeMillis();
 
-            if (_isPreserveUser && _fileSelection != FileSelection.RECURSE) {
+            if (_isPreserveUser && !_isNumericIds
+                    && _fileSelection != FileSelection.RECURSE) {
                 sendUserList();
```

Comment:
I fixed this problem with commit https://github.com/usrflo/yajsync/commit/93d508d4f2be03e1895a7b0323baad12e8666139 but failed to squash the two commits of this branch (maybe you can add these 3 lines in your submit directly).